db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
```

Comment:
Can we remove "Express" and just have FedEx logo? Adding Express can limit uses when it mig
![CleanShot 2024-02-02 at 12 39 15@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/c08ead23-661f-4460-b6c4-6d5bd5f87838)
ht be ground/overnight/freight.